### PR TITLE
(OSCI) - Add author Arnav Grover

### PR DESCRIPTION
**Pull request by Arnav-Gr0ver**
### Description
Adds author Arnav Grover for OSCI 2023
 
### Check List

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/arnavgrover.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Arnav Grover
+short_name:
+twitter:
```
Comment (dtaivpp) [inline _authors/arnavgrover.markdown:4]: Please remove the twitter tag and recommit and we should be good!

**Review by dtaivpp — CHANGES_REQUESTED** [review]

File: _authors/arnavgrover.markdown (hunk 1 above)
Comment (Arnav-Gr0ver) [reply _authors/arnavgrover.markdown]: Just recommitted!

**Comment by dtaivpp** [comment]
Ah sorry I missed one other thing, can you move your bio from the `_authors` to the `_authors/osci_2023`. We are so close and thanks for all the work!

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**